index.html

```diff
@@ -5,523 +5,1137 @@
-            their work.
-          </p>
-          <img src="/" alt="Samuel Treavor's profile photo" />
-          <h4>Samuel Treavor</h4>
-          <p class="rating">&starf; &starf; &starf; &starf; &starf;</p>
-          <button type="button">&lt;</button>
-          <button type="button">&gt;</button>
-        </article>
+        <div class="container testimonials-bg"></div>
+        <div class="testimonials-wrapper container">
+          <div class="testimonials-content">
+            <h2 class="section-name franklin-normal-16">TESTIMONIALS</h2>
+            <h3 class="section-title">
+              <span class="header-line">Hear from Our </span>
+              <span class="header-line">Customers</span>
+            </h3>
+          </div>
+          <article class="testimonials-card">
+            <p class="testimonials-card-text">
+              Discovering PixMart's digital design assets and resources
+              revolutionized my client projects. The quality and variety have
+              allowed me to create stunning visuals that captivate my audience
+              every time. Highly recommended for any creatives looking to
+              elevate their work.
+            </p>
+            <div class="testimonials-card-details">
+              <div class="testimonials-card-name">
+                <img
+                  src="./img/img-blog-homepage/user-photo.webp"
+                  alt="Samuel Treavor's profile photo"
+                  width="64"
+                  height="64"
+                />
+                <div class="testimonials-rating">
+                  <h4 class="testimonials-card-title franklin-bold-16">
+                    Samuel Treavor
+                  </h4>
+                </div>
+              </div>
+              <div class="testimonials-btn-group">
+                <button type="button" class="btn-arrow">
+                  <svg class="icon arrow-less" width="16" height="16">
+                    <use xlink:href="./img/sprite.svg#icon-arrow-down"></use>
+                  </svg>
+                </button>
+                <button type="button" class="btn-arrow">
+                  <svg class="icon arrow-greater" width="16" height="16">
+                    <use xlink:href="./img/sprite.svg#icon-arrow-down"></use>
+                  </svg>
+                </button>
+              </div>
+            </div>
+          </article>
+        </div>
       </section>
 
       <!-- Pricing section -->
-      <section class="pricing">
-        <h2 class="section-name">PRICING PLAN</h2>
+      <section class="pricing container">
+        <h2 class="section-name franklin-normal-16">PRICING PLAN</h2>
         <h3 class="section-title">
           Choose The Perfect Plan for Creative Potential
         </h3>
 
-        <ul>
-          <li>
+        <ul class="membership-plan-list">
+          <li class="membership-plan-item">
             <!-- Price Plan -->
             <article class="membership-plan">
```

Comment:
No need extra wrapper `membership-plan` - you can move styles to `membership-plan-item` and set additional class for central element

Reply:
I'm not sure about this solution.. If you want me to reduce number of wrappers, I could make a div with articles.. 

Please let me know your thoughts about this matter.

#need-clarification

Reply:
I have fixed both price cards and product cards. I also asked Liza about this case, she said both  yours and mine options are valid. 
As for product cards, I marked up them without list but previous reviewer said I should put all articles in lists. 
So it's a bit frustrating.

Reply:
Probably because there is no single rule on how to do it correctly. I proceeded from the point that it is worth removing extra HTML